### PR TITLE
[16.0][FIX] account_payment_partner: fix error in _reverse_moves

### PR DESCRIPTION
**Pull request by ramiadavid**

### CODE REVIEW

File: account_payment_partner/models/account_move.py [hunk 1]
```diff
@@ -94,6 +94,9 @@ def _compute_partner_bank_id(self):
         return res
 
     def _reverse_moves(self, default_values_list=None, cancel=False):
+        if not default_values_list:
+            default_values_list = [{} for _ in self]
+
```
Comment (pedrobaeza) [inline account_payment_partner/models/account_move.py]: Don't add empty lines.

**Review by pedrobaeza — COMMENTED** [review]
Can you provide steps to reproduce the problem? Why it would get this method an empty `default_values_list` content?

**Comment by ramiadavid** [comment]
Hi @pedrobaeza 

An error occurs when running tests on this file: https://github.com/OCA/l10n-spain/blob/483e5c7adb3e34948067df6aab7b9d779d888eb2/l10n_es_aeat/tests/test_l10n_es_aeat_taxinfo.py#L26

All the official addons that include this function have that code that I have added.
For example:
https://github.com/odoo/enterprise/blob/1d4432f6394d68f2e2d37d98babece555ae1c9ff/account_asset/models/account_move.py#L122
https://github.com/odoo/enterprise/blob/1d4432f6394d68f2e2d37d98babece555ae1c9ff/partner_commission/models/account_move.py#L150

File: account_payment_partner/models/account_move.py (hunk 1 above)
Comment (ramiadavid) [reply account_payment_partner/models/account_move.py]: done

**Review by pedrobaeza — APPROVED** [review]
I don't have access to the enterprise repo, but I trust you, so let's merge this:

/ocabot merge patch

Thanks for the patch

**Pull request merged**